src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -112,11 +133,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $io->title(sprintf('Current configuration for "%s.%s"', $extensionAlias, $path));
 
-        $io->writeln(Yaml::dump($config, 10));
+        $io->writeln($this->convertToFormat($config, $format));
 
         return 0;
     }
 
+    private function convertToFormat(mixed $config, string $format): string
+    {
+        return match ($format) {
+            'yaml' => Yaml::dump($config, 10),
+            'json' => json_encode($config, \JSON_PRETTY_PRINT),
```

Comment:
Add more options for good looking JSON.
```suggestion
            'json' => json_encode($config, \JSON_PRETTY_PRINT|\ JSON_UNESCAPED_SLASHES|\JSON_UNESCAPED_UNICODE),
```